OpenTabletDriver.Configurations/Parsers/Veikk/VeikkTiltedTabletReport.cs

```diff
@@ -0,0 +1,37 @@
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using OpenTabletDriver.Tablet;
+
+namespace OpenTabletDriver.Configurations.Parsers.Veikk
+{
+    public struct VeikkTiltedTabletReport : ITabletReport, ITiltReport
+    {
+        public VeikkTiltedTabletReport(byte[] report)
+        {
+            Raw = report;
+
+            Position = new Vector2
+            {
+                X = Unsafe.ReadUnaligned<ushort>(ref report[3]) | (report[5] << 16),
+                Y = Unsafe.ReadUnaligned<ushort>(ref report[6]) | (report[8] << 16)
+            };
+            Tilt = new Vector2
+            {
+                X = unchecked((sbyte)report[11]),
+                Y = unchecked((sbyte)report[12])

```

Comment:
I don't think unchecked is needed here as this is a same-length cast.
```suggestion
                X = (sbyte)report[11],
                Y = (sbyte)report[12]

```

Reply:
well actually it's effects how it's works. if i remove unchecked it will throw error.
`Constant value '128' cannot be converted to a 'sbyte' (use 'unchecked' syntax to override)`

otherwise i need to use formula but I'm not math guy so if I can achieve result simpler way I do.
because of that only part i can't touch.